CLIENT-SPECIFICATION.md

```diff
@@ -85,7 +85,7 @@ tldr --platform osx bash
 
 This section documents the directory structure that contains the pages themselves.
 
-The master version of every page is stored inside (but not directly) the `pages` directory. Inside this directory, there is a folder for each platform - for example `windows`, `linux`, and the special `common` platform:
+The main version of every page is stored inside (but not directly) the `pages` directory. Inside this directory, there is a folder for each platform - for example `windows`, `linux`, and the special `common` platform:
```

Comment:
Should the pages in the `pages` directory be in English?

Reply:
Not exactly sure what you mean, but I added a clarification that `pages/` is in English (outside of the scope of this PR).

Reply:
A question for reviewers, this `md` assume the `pages/` dir is only for English pages as reference for other languages for translation.